index.js

```diff
@@ -26,16 +25,32 @@ export {
 };
 
 let loaded;
-export async function loadOnce(mediatorUrl = DEFAULT_MEDIATOR) {
+export async function loadOnce(options) {
   if(loaded) {
     return loaded;
   }
 
   loaded = true;
-  return load(mediatorUrl);
+  return load(options);
 }
 
-export async function load(mediatorUrl = DEFAULT_MEDIATOR) {
+export async function load(options = {
+  mediatorOrigin: DEFAULT_MEDIATOR_ORIGIN
+}) {
+  // backwards compatibility (`options` used to be a string for expressing
+  // the full mediator URL)
+  let mediatorUrl;
+  if(typeof options === 'string') {
+    mediatorUrl = options;
+  } else if(options && typeof options === 'object' &&
+    typeof options.mediatorOrigin === 'string') {
+    mediatorUrl = `${options.mediatorOrigin}/mediator`;
+  } else {
+    throw new Error(
+      '"options.mediatorOrigin" must be a string express the ' +
```

Comment:
Should that be "expressing"?  Might just reword to say "options.mediatorOrigin" is missing.

Reply:
I read the code wrong, maybe it can just say that param needs to be a string.

Reply:
I think it's better to be more descriptive here. I fixed the typo.